db/fixtures/miq_product_features.yml

```diff
@@ -392,57 +392,61 @@
   :feature_type: node
   :identifier: service
   :children:
-  - :name: All Services
-    :description: Everything under All Services Accordion
-    :feature_type: node
-    :identifier: service_accord
+  - :name: View All Services
+    :description: View Services
+    :feature_type: view
+    :identifier: service_view
     :children:
```

Comment:
I think we should keep a top level.

a top level of `services' (model name) seems a little short but I see `_explorer` and that seems good

/cc @Fryguy you have an opinion here?


```yaml
  - :name: All Services
    :description: Everything under All Services
    :feature_type: node
    :identifier: service_explorer
    :children:
```

Reply:
trying again

the top level `service` has nothing else underneath it.
Delete service_accord

in a migration replace all `service_accord` privileges with `service`

In the migration, do not worry about the service_show_list